services/pdf-ms/helpers/user.ts

```diff
@@ -0,0 +1,22 @@
+import to from 'await-to-js';
+
+import config from '../../../config';
+import * as dynamoDb from '../../../libs/dynamoDb';
+
+export const getUser = async (personalNumber: string) => {
+  const params = {
+    TableName: config.users.tableName,
+    Key: {
+      personalNumber,
+    },
+  };
+  const [error, userGetResponse] = await to<{ Item: Record<string, any> }>(
```

Comment:
getUsersDbResponse är trevligare att uttala. Eller?
Ännu bättre är bara usersDbResponse
Passar fint med usersDbResponse.Item

Reply:
Renaming to usersDbResponse.